inc/import.class.php

```diff
@@ -0,0 +1,1220 @@
+            } else if ($itemType == "PluginResourcesContractType") {
+               if ($datas["origin"] == "Externe") {
+                  $options["value"] = $findVal['id'];
+               }
+            } else {
+               $options["value"] = $findVal['id'];
+            }
+
+         }
+      }
+      $options["name"] = "resource[values][" . $datas['id'] . "][$sqlField]";
+      Dropdown::show($itemType, $options);
+      echo "</td>";
+   }
+
+   /**
+    * @param $fieldsToCheck
+    * @param $datas
+    * @param $field
+    *
+    * @return string
+    */
+   function ShowDiffField($fieldsToCheck, $datas, $field) {
+      foreach ($fieldsToCheck as $fieldImport => $fieldResource) {
+         if ($datas[$fieldResource] != $datas[$fieldImport] &&
+             ($field == $fieldResource || $field == $fieldImport)) {
+            $showDiff[$fieldResource]['showDiff'] = "font-weight:bold;";
+            $showDiff[$fieldImport]['showDiff']   = "font-weight:bold;";
+         } else {
+            $showDiff[$fieldResource]['showDiff'] = "";
+            $showDiff[$fieldImport]['showDiff']   = "";
+         }
+      }
+      return $showDiff;
+   }
+
+   /**
+    * Init header list
+    */
+   function listHead() {
+      global $CFG_GLPI;
+      echo "<form name='form' method='post'
+         action ='" . $CFG_GLPI["root_doc"] . "/plugins/resources/front/import.php?actionImport=" . $_SESSION['actionImport'] . "' >";
+      echo "<input type='hidden' name='actionImport' value='" . $_SESSION['actionImport'] . "'>";
+      echo "<div align='center'>
+                <table boreder='0' class='tab_cadrehov'>";
+
+
+      $entete = 0;
+      if ($_SESSION['actionImport'] == "checkIncoherences") {
+         echo "<tr>";
+         echo "<th colspan='12'>Données Fichier External</th>";
+         echo "<th style='border-left:2px solid black' colspan='8'>Données ressource GLPI</th>";
+         echo "</tr>";
+      }
+      echo "<tr>";
+      echo "<th class=''>
+                <div class='form-group-checkbox'>
+                   <input title='Tout cocher comme' type='checkbox' class='new_checkbox' name='checkall_imports' id='checkall_imports' 
+                   onclick='checkAll(this.checked);' >";
+      echo "<script>
+      function checkAll(state) {
+         var cases = document.getElementsByTagName('input');
+         for(var i=0; i<cases.length; i++){
+           if(cases[i].type == 'checkbox'){
+                cases[i].checked = state;   
+            } 
+         }
+     }       
+      </script>";
+      echo "<label class='label-checkbox' for='checkall_imports' title='Tout cocher comme'>
+                      <span class='check'></span>
+                      <span class='box'></span>
+                   </label>
+                </div>
+              </th>";
+      echo "<th>" . __("External ID", 'resources') . "</th>";
+      echo "<th>" . __("Registration number", 'resources') . "</th>";
```

Comment:
Use __('Administrative number') in fr_FR.po : 
msgid "Administrative number"
msgstr "Matricule"